crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -341,10 +322,7 @@ impl<Codec: RawEntryCodec> StateMachine<Codec> {
 
         Span::record_invocation_id(&invocation_id);
         Span::record_invocation_target(&service_invocation.invocation_target);
-        service_invocation
-            .span_context
-            .as_parent()
-            .attach_to_span(&Span::current());
+        Span::current().set_relation(service_invocation.span_context.as_parent());
```

Comment:
I think it would be a great state if we can send the runtime and services traces to separate jaeger endpoints and the traces on each of the endpoints are consistent and helpful. Whether they link together when shown on the same endpoint, is probably not so important because the services traces do only present a make-belief reality anyway.